include/eigenpy/quaternion.hpp

```diff
@@ -263,24 +268,42 @@ namespace eigenpy
       Quaternion* q(new Quaternion); q->setIdentity();
       return q;
     }
-    
-    static Quaternion* FromTwoVectors(const Vector3& u, const Vector3& v)
+
+    static Quaternion* FromCoefficients(Scalar w, Scalar x, Scalar y, Scalar z)
+    {
+      Quaternion* q(new Quaternion(w, x, y, z));
+      return q;
+    }
+
+    static Quaternion* FromAngleAxis(const AngleAxis& aa)
```

Comment:
Note, this won't compile with `Eigen::Ref<AngleAxis>`

Reply:
Why do you want to rely on `Eigen::Ref<AngleAxis>`?

Reply:
Right, it doesn't matter here. `const Eigen::Ref<const TYPE>&` is my first approach out of habit